src/Voron/Data/BTrees/Tree.cs

```diff
@@ -365,8 +365,14 @@ public DirectAddScope DirectAdd(Slice key, int len, out byte* ptr)
         {
             return DirectAdd(key, len, TreeNodeFlags.Data, out ptr);
         }
+        
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public DirectAddScope DataDirectAdd(Slice key, int len)
```

Comment:
1. Shouldn't be rather `AddKey()` / `AddKeyOnly()`?
2. Do you need `int len` parameter? Isn't it always 0?
3. Can we make it `internal` instead of public since it's used for multi add purposes only